day_7/checker_board.rb

```diff
@@ -0,0 +1,20 @@
+size_of_board = 10
+
```

Comment:
A next step with some of these exercises could be to leverage gets.chomp to run these based on user input.